source/client/module.coffee

```diff
@@ -1,26 +1,35 @@
-class Space.accountsUi extends Space.Module
+Space.accountsUi = Space.Module.define 'Space.accountsUi',
 
-  @publish this, 'Space.accountsUi'
+  configuration: {
```

Comment:
:+1: me too, that was the hardest part to get right. I had deep thoughts about that before coming up with this pattern.